video_live_generator.py

```diff
@@ -9,8 +9,8 @@ def __init__(self, input_port):
         self.input = cv2.VideoCapture(int(input_port))
         self.set_camera_settings(str(input_port))
 
-        self.input.set(cv2.CAP_PROP_FRAME_WIDTH, 640)
-        self.input.set(cv2.CAP_PROP_FRAME_HEIGHT, 480)
+        self.input.set(cv2.CAP_PROP_FRAME_WIDTH, 1080)
```

Comment:
1080p refers to the number of vertical lines, see https://en.wikipedia.org/wiki/1080p